i18n/en.json

```diff
@@ -2203,6 +2203,7 @@
   "edit_channel_header_modal.title": "Edit Header for {channel}",
   "edit_channel_header_modal.title_dm": "Edit Header",
   "edit_channel_header.editHeader": "Edit the Channel Header...",
+  "channel_header.editLink": "(Edit)",
```

Comment:
This is not in the place that is expected (it must be alphabetically ordered by id in the file).